tests/integration/features/backup_scheduled_pitr_psmdb/10-create-cluster.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
   name: test-psmdb-cluster
 spec:
   engine:
+    # in psmdb 1.20 the pbm initialization process has changed. So first let's test the backups creation for older CR versions
+    crVersion: 1.19.1
```

Comment:
the same

Reply:
added the initial CR version patch as a separate step to be able to use a variable